customskill/examples/http.go

```diff
@@ -36,10 +36,9 @@ func main() {
 	http.ListenAndServe(":8080", nil)
 }
 
-func onLaunch(launchRequest *request.LaunchRequest) (*response.Response, map[string]interface{}, error) {
+func onLaunch(launchRequest *request.LaunchRequest, metadata *request.Metadata) (*response.Response, map[string]interface{}, error) {
 	resp := response.New()
 	resp.SetEndSession(response.Bool(true))
```

Comment:
Should the session stay open if the user is launching the skill? seems like the example could be misleading.

Reply:
Would it be worth adding some speech here so if the user tries it out then they can at least get some output speech?

Reply:
Seems reasonably. It's on my "todo" list but I'm currently getting slammed at work. Hope to have time later this week to work on this.